<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Session Students Detail</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="titleLBL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>File Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLBL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Date : </string>
    </property>
   </widget>
   <widget class="QLabel" name="startTimeLBL">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Waree</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start Time : </string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="analysisBTN">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Display</string>
   </property>
  </widget>
  <widget class="QLabel" name="msgLBL">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu Condensed</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>731</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yrsa</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="usesScrollButtons">
    <bool>true</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="recogBTN">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analyse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
